frontend/kesaseteli/employer/src/pages/login.tsx

```diff
@@ -1,25 +1,52 @@
+import { IconSignin } from 'hds-react';
+import useLogin from 'kesaseteli/employer/hooks/backend/useLogin';
 import { GetStaticProps, NextPage } from 'next';
 import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
 import * as React from 'react';
 import Container from 'shared/components/container/Container';
 import withoutAuth from 'shared/components/hocs/withoutAuth';
 import Layout from 'shared/components/Layout';
 import useClearQueryParams from 'shared/hooks/useClearQueryParams';
 import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
 
+import { $Notification, $PrimaryButton } from '../components/application/login.sc';
+
 const Login: NextPage = () => {
   useClearQueryParams();
+  const { t } = useTranslation();
   const {
     query: { logout, error, sessionExpired },
   } = useRouter();
+  const login = useLogin();
+
+  let notificationLabel = t(`common:loginPage.infoLabel`)
```

Comment:
ususally using `let` should be avoided, perhaps we could use method here: `const notificationKey = getNotificationLabelKey(query)` ans put the logic into method. then use it `t(notificationKey)`